From 44afeafb8847a8487940cf1f3480108db51e2b41 Mon Sep 17 00:00:00 2001
From: Jan Kara <jack@suse.cz>
Date: Wed, 29 Jul 2026 15:35:31 +0200
Subject: [PATCH 08236/15333] fsnotify: Remove Matt Bobrowski as a reviewer

Matt was not very active reviewer in last years. Furthermore he
apparently left Google and his email doesn't work anymore. I didn't find
any working contact for him so at least for now delete the entry with
non-existent email.

Signed-off-by: Jan Kara <jack@suse.cz>
---
 MAINTAINERS | 1 -
 1 file changed, 1 deletion(-)

diff --git a/MAINTAINERS b/MAINTAINERS
index 15011f5752a9..a93e2cac04cd 100644
--- a/MAINTAINERS
+++ b/MAINTAINERS
@@ -9907,7 +9907,6 @@ F:	net/core/failover.c
 FANOTIFY
 M:	Jan Kara <jack@suse.cz>
 R:	Amir Goldstein <amir73il@gmail.com>
-R:	Matthew Bobrowski <repnop@google.com>
 L:	linux-fsdevel@vger.kernel.org
 S:	Maintained
 F:	fs/notify/fanotify/
-- 
2.55.0

